Services/Interfaces/web_interface/static/js/components/dashboard.js

```diff
@@ -48,8 +48,6 @@ $(document).ready(function () {
 
     const updateProfitabilityDisplay = (bot_real_profitability, bot_real_flat_profitability,
                                         bot_simulated_profitability, bot_simulated_flat_profitability) => {
-        bot_simulated_profitability = 21
-        bot_simulated_flat_profitability = 190
```

Comment:
😮

Reply:
it's not released, I accidentally added it recently

Reply:
👍